src/components/landing-tree-status/landing-tree-stats.less

```diff
@@ -0,0 +1,15 @@
+.dark-green {
+  color: #61802e;
+}
+
+.tree-stats-container {
+  margin-top: 35px;
+}
+
+.map-cards-container {
+  margin: 30px auto;
+}
+
+.map-card {
```

Comment:
What if the MapCard itself had a margin?

Reply:
You were right about MapCard not being a great name, it's used on the team leaderboard with a different margin so I figured it made more sense for the card to not have any margins of its own.